src/Data/provideLocalStorageDataClass.ts

```diff
@@ -10,6 +10,8 @@ interface RawDataItem {
   [key: string]: unknown
 }
 
+const recordKeys: Set<{ recordKey: string; className: string }> = new Set()
```

Comment:
* [ ] FYI Making this a Set is misleading b/c the seemingly same key, e.g. `recordKeys.add({ recordKey: "foo", className: "bar" })` will be added multiple times to the set. How about making it an Array instead?